gen3-integration-tests/services/requestor.py

```diff
@@ -62,6 +36,62 @@ def create_request(self, data, token):
 
         return request_id
 
+    def create_request_with_authHeader(
+        self,
+        user="main_account",
+        username=None,
+        policy_id=None,
+        resource_paths=None,
+        role_ids=None,
+        revoke=False,
+        request_status=None,
+    ):
+        """Create a new request in requestor with auth_headers"""
+        data = {}
+        gen3auth = Gen3Auth(refresh_token=pytest.api_keys[user])
+        # if policy_id is argument
+        if policy_id and not (resource_paths or role_ids):
+            logger.info(
+                f"Creating request with policy_id : {policy_id} with revoke {revoke}"
+            )
+            data = {
+                "policy_id": policy_id,
+                "username": username,
+            }
+        # if resource_paths and role_ids is arguments
+        elif resource_paths and role_ids and not policy_id:
+            logger.info(
+                f"Creating request for resource_paths : {resource_paths} and roles_ids : {role_ids} with revoke {revoke}"
+            )
+            data = {
+                "username": username,
+                "resource_paths": resource_paths,
+                "role_ids": role_ids,
+            }
+        else:
+            logger.info(
+                f"Incorrect args in create request: must have policyID or resourcePaths+roleIds"
+            )
+            return None
+        # if request_status is argument
+        if request_status:
+            data["status"] = request_status
+        # if revoke=true using revoke url else using base_url
+        endpoint = f"{self.REVOKE_URL}" if revoke else f"{self.BASE_URL}"
+        logger.info(f"endpoint: %s", endpoint)
+        # send post request
+        create_req = requests.post(
+            endpoint,
+            json=data,
+            headers={"Authorization": f"bearer {gen3auth._access_token}"},
```

Comment:
we shouldn't be using private methods, there should be a get_access_token that we can use